solutions/security-enforced/variables.tf

```diff
@@ -65,9 +65,9 @@ variable "service_plan" {
   }
 }
 
-variable "skip_sm_ce_iam_authorization_policy" {
+variable "skip_secrets_manager_certificate_engine_iam_auth_policy" {
```

Comment:
certificate engine is not correct here. This boolean controls the IAM credentials engine